var/spack/repos/builtin/packages/sirius/package.py

```diff
@@ -162,18 +171,38 @@ class Sirius(CMakePackage, CudaPackage, ROCmPackage):
         depends_on("dla-future +cuda", when="+cuda")
         depends_on("dla-future +rocm", when="+rocm")
 
+        conflicts("^pika@:0.22.1", when="+cuda")
+        conflicts("^pika@:0.22.1", when="+rocm")
+
     depends_on("rocblas", when="+rocm")
     depends_on("rocsolver", when="@7.5.0: +rocm")
 
+    # FindHIP cmake script only works for < 4.1, but HIP 4.1 is not provided by spack anymore
+    conflicts("+rocm", when="@:7.2.0")
+
     conflicts("^libxc@5.0.0")  # known to produce incorrect results
     conflicts("+single_precision", when="@:7.2.4")
     conflicts("+scalapack", when="^cray-libsci")
 
     # Propagate openmp to blas
-    depends_on("openblas threads=openmp", when="+openmp ^[virtuals=blas] openblas")
```

Comment:
Not sure what is correct here, `virtuals=blas` or `virtuals=blas,lapack`?

Reply:
Can this conversation be resolved? Or are more changes needed before the merge?

Reply:
I changed it from `[virtuals=blas]` to `[virtuals=blas,lapack]`, because imo blas and lapack provider should be the same.
If nobody is against this change, we can merge it.

Reply:
`openblas` does indeed provide `blas` and `lapack` together: https://github.com/spack/spack/blob/b1d4fd14bc99c8c6d270bdadf897dbaf6ffc6b40/var/spack/repos/builtin/packages/openblas/package.py#L100